capabilityquerier/pkg/capabilityquerier/capabilityquerier_test.go

```diff
@@ -58,14 +60,18 @@ func Test_GetAndSendCapabilityStatement(t *testing.T) {
 	}
 	err = json.Unmarshal(expectedCapStat, &(expectedMsgStruct.CapabilityStatement))
 	th.Assert(t, err == nil, err)
-	expectedMsg, err := json.Marshal(expectedMsgStruct)
+	expectedMsg, err := json.Marshal(expectedMsgStruct.CapabilityStatement)
 	th.Assert(t, err == nil, err)
 
 	// execute tested function
 	err = GetAndSendCapabilityStatement(ctx, fhirURL, &(tc.Client), &mq, &ch, queueName)
 	th.Assert(t, err == nil, err)
 	th.Assert(t, len(mq.(*mock.BasicMockMessageQueue).Queue) == 1, "expect one message on the queue")
 	message = <-mq.(*mock.BasicMockMessageQueue).Queue
+
+	var msgJSON map[string]interface{}
+	_ = json.Unmarshal(message, &msgJSON)
+	message, _ = json.Marshal(msgJSON["capabilityStatement"])
```

Comment:
I think instead of just testing the Capability Statement, the SMARTHTTPResponse and the SMARTResp should be added to the `expectedMsgStruct` and the whole message should be compared

Reply:
In Test_GetAndSendCapabilityStatement it uses a specific test client from testClientWithContentType. It seems like it doesn't actually query it but gets the results from the testdata/.. directory. Should I do something similar and add a response statement from the well known endpoint since it doesn't actually get one?

Reply:
oh hmm okay, yeah that seems reasonable to me!